config.sample.yml

```diff
@@ -91,6 +91,13 @@ logging:
   # (Optional) Logging settings. You can have a severity debug,info,warn,error
   #
   level: info
+permissions:
+  # (Optional) Permissions for using the bridge
+  #
+  - actor: example.com
+    services:
+      - service: "*"
+        level: admin
```

Comment:
need to document possible values and meanings

Reply:
See setup.md